recipes/default.rb

```diff
@@ -47,7 +45,21 @@
   cwd "#{node['timesync']['application_path']}/source"
 end
 
+execute 'create root user' do
+  command "npm run create-account -- -u root -p #{secret_key['root_pass']}"
+  environment environment
+  sensitive true
+  cwd "#{node['timesync']['application_path']}/source"
+end
```

Comment:
The create-account script silently and immediately exits if any accounts already exist in the database for exactly that reason. It shouldn't add any significant overhead to subsequent runs.